src/main/java/org/opentripplanner/profile/StopTreeCache.java

```diff
@@ -28,35 +27,72 @@
 public class StopTreeCache {
```

Comment:
Needs javadoc, perhaps borrowing some from the propagateStop function. Will be done after the merge.